data/moves.js

```diff
@@ -11142,7 +11142,7 @@ exports.BattleMovedex = {
 		name: "Rain Dance",
 		pp: 5,
 		priority: 0,
-		flags: {},
+		flags: {dance: 1},
```

Comment:
That may want to be checked in actual Sun & Moon, because in Japanese version, this move is called Rain Prayer (and I believe Japanese version matters).